src/main/java/Deadline.java

```diff
@@ -0,0 +1,19 @@
+public class Deadline extends Todo{
```

Comment:
For your overall code, I think it would be more readable if you could add in comments at the top of each method/function.